container/test/worker_wrappers.yaml

```diff
@@ -15,7 +15,7 @@ fileExistenceTests:
   - name: "as-nobody wrapper"
     path: '/app/build_buildfarm/as-nobody'
     shouldExist: true
-    permissions: '-r-xr-xr-x'
+    permissions: '-rwxr-xr-x'
```

Comment:
Why is this necessary for this change?

Reply:
This just seems to be how `rules_pkg` is delivering this particular file. I can try to explicitly set it to `-r-xr-xr-x`

Reply:
There's really no reason for any file deployed under the buildfarm container root as an application file to have writable permissions, at all. Try to drop this if possible

Reply:
Fixed!